ceph/rados/crushtool_workflows.py

```diff
@@ -667,3 +667,122 @@ def add_crush_rule(self, rule_name: str, rules: str) -> bool:
             return False
         log.info(f"Successfully set rules with name {rule_name} on the cluster")
         return True
+
+
+class OsdToolWorkflows:
```

Comment:
Can we consider separating this class in another library file meant for osdmaptool workflows?

Reply:
We can. Will take it up only if necessary.

Reply:
Will take it up later